src/languageservice/utils/arrUtils.ts

```diff
@@ -73,7 +73,7 @@ export function filterInvalidCustomTags(customTags: string[]): string[] {
   });
 }
 export function isArrayEqual(fst: Array<unknown>, snd: Array<unknown>): boolean {
-  if (!snd) {
+  if (!snd || !fst) {
```

Comment:
Is this being tested? I don't see this function being called in the test.

NOT logic is easy to get wrong and cause unforeseen issues. It is better to test it.

Reply:
This is tested but not directly. I just found that `isArrayEqual` not handle situation when first argument is `undefined` or `null`. I do not think that providing test for  `arrUtils.ts` is a subject for this PR.

Reply:
I approved it. 
It is better to keep changes out of a commit / PR that do not directly address the issue. They should have their own commit/PR. If you include it then testing it is expected too.